[GW2C-17] Add CONTRIBUTING.md (#3)
lazybytez/Guild-Wars-2-Zone: CONTRIBUTING.md

# Contributing

> Some of you may not even know git so we try to explay it to your as clear as possible what we need to accept your contribution:

Reviewer: Please fix the typo
```suggestion
Some of you may not even know git so we try to explay it to you as clear as possible what we need to accept your contribution:
```

## Look first before you Contribute

> Search through the issues. Maybe somebody has already created an issue with your same intention in mind. If not create one yourself and comment that you'll work on it.  
> (not necessary for typo or grammar fixes)

Reviewer: This sentence can be optimized:
```suggestion
Search through the issues. Maybe somebody has already created an issue with the same intention in mind. If not, create an issue by yourself and comment that you'll work on it.  
(not necessary for typo or grammar fixes)
```

## Commit messages

> Commit messages in **english** and **imperative**. It should be a **short description** of your change.  
> Different types of changes require different prefixes for example:

Reviewer: I would suggest the following wording:
```suggestion
Commit messages should be written in **english** and use the **imperative** form. It should be a **short description** of your change.  
Different types of changes require different prefixes for example:
```

- if you added / changed or removed content:
```
docs: make a content change of farming-guides.md 
```
- if you found a small mistake and want to fix it:
```
fix: correct a typo and grammatical error in farming-guides.md
```
- if you edit something which is unrelated to `/content` and from the repository itself:
```
repo: edit the README.md 
```

### Writing the article
Nobody has written a guide about your idea yet? Then you are free to go and **write your own**: 

> 1. Be sure to write article meta data. Ex:

Reviewer: I would suggest the following wording:
```suggestion
1. Be sure to add the article meta data. Ex:
```

```
---
title: Example article title
description: An article to show you the markdown we support currently!
thumbnail: general-category-name/more-specified-image-name.jpg
authors:
  - KompetenzElle.6431
---
```

- The title will be shown as the article title on the **sidebar** and **main page**.
- The description should be a small description for your article and is also shown on the **main page**.
- A thumbnail would be appreciated for a more colorful user experience. **Always** take an image with a **copyright free license** or from the Game itself. We don't want any **license issues**!!!  

> If you have found a copyright free image, create or place it into a a category. "All images related to dragonfall farming into `images/dragonfall` etc..."  

Reviewer: We should tell that the images folder is in the assets directory:
```suggestion
If you have found a copyright free image, create or place it into a a category. "All images related to dragonfall farming into `assets/images/dragonfall` etc..."  
```

The name should also make a little bit of sense and not just be obfuscated characters. 

> - This is optional **but** because this is about `Guild Wars 2`, you should your ingame name if someone has questions or want to praise your work :) 

Reviewer: There is a word missing and the sentence could be optimized:
```suggestion
- This is optional **but** as this is about `Guild Wars 2`, you should provide your ingame name if someone has questions or want to praise your work :) 
```
